math/minuit2/src/Minuit2Minimizer.cxx

```diff
@@ -421,11 +426,11 @@ bool Minuit2Minimizer::Minimize() {
    }
 
    // internal minuit messages
-   MnPrint::SetLevel(printLevel );
    fMinimizer->Builder().SetPrintLevel(printLevel);
```

Comment:
Is this line still needed ? Maybe not, since we use later  MnPrint::SetGlobalLevel

Reply:
I am not sure, I was a bit confused by the code in Minuit2Minimizer, so I tried to change as little as possible.

A bit off-topic, but do you agree that ultimately our goal is to allow each class to have its own printing level that can be set by users, and to only fall back to a global print level if no custom setting is used?